mail-storage/src/main/java/mail/storage/domain/Message.java

```diff
@@ -0,0 +1,32 @@
+package mail.storage.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.index.Indexed;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.io.Serializable;
+import java.util.Date;
+
+@Document("message")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class Message implements Serializable {
+    @Id
+    private String id;
+    @Indexed(unique = true)
+    private Long number;
+    private String sender;
+    private String receiver;
+    private String topic;
+    private String body;
+    private Date date;
+    private String attachmentUrl;
+
+    private MessageType type;
```

Comment:
it's better to encapsulate this enum in Message, because the enum can't exist without the entity

Reply:
Fixed.